recipes/install.rb

```diff
@@ -40,8 +49,13 @@
     enabled node['topbeat']['yum']['enabled']
     action node['topbeat']['yum']['action']
   end
+
+  package 'yum-versionlock'
+  execute 'yum versionlock clear topbeat'
+  execute "yum versionlock topbeat-#{version_string}"
```

Comment:
@pmorton Could we make **execute** resources conditional or better use [yum-plugin-versionlock](https://supermarket.chef.io/cookbooks/yum-plugin-versionlock) cookbook?